azkaban_cli/azkaban_cli.py

```diff
@@ -221,6 +221,44 @@ def __fetch_projects(ctx, user):
     except FetchProjectsError as e:
         logging.error(str(e))
 
+@login_required
+def __add_permission(ctx, project, group, admin, read, write, _execute, _schedule):
+    azkaban = ctx.obj[u'azkaban']
+    try:
+        azkaban.add_permission(
+            project, 
+            group, 
+            permission_options= {
+                'admin':admin, 'read':read, 'write':write, 'execute':_execute, 'schedule': _schedule
+            }
+        )
+    except FetchProjectsError as e:
+        logging.error(str(e))
+
+@login_required
+def __remove_permission(ctx, project, group):
+    azkaban = ctx.obj[u'azkaban']
+    try:
+        azkaban.remove_permission(
+            project, 
+            group
+        )
+    except FetchProjectsError as e:
+        logging.error(str(e))
+
+@login_required
+def __change_permission(ctx, project, group, admin, read, write, _execute, _schedule):
```

Comment:
Yes, I agree and I'm sure it can be done in a way to DRY

But looking in the code all CLI commands have a function, doing exactly what these two are doing, calling another function. 

Looking in another way, I believe these functions are isolated to represent a view of the CLI, separating its responsibility from the "controller" and other components.